### PR TITLE
Homework14

### PR DESCRIPTION
**Pull request by SergeyLinnickHub**
Have one unresolved error http://prntscr.com/ngpz8r

### CODE REVIEW

File: src/components/numbers/Numbers.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import './numbers.scss';
+
+export const Numbers = ({
+  from, to, odd, even
+}) => {
+  const renderItems = () => {
+    const li = [];
+    for (let i = from; i <= to; i++) {
+      (odd && (i % 2 === 0) || even && (i % 2 === 1) || !even && !odd) && (
```
Comment (iliatcymbal) [inline src/components/numbers/Numbers.js]: use if here
```
if (odd && (i % 2 === 0) || even && (i % 2 === 1) || !even && !odd) { ... }
```

File: src/components/greeting/Greeting.js [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+export const Greeting = ({ name }) => (
+  <h2>
+    {calculateGreeting()}
+    {' '}
+    {name}
+
+
+!
+  </h2>
+);
```
Comment (iliatcymbal) [inline src/components/greeting/Greeting.js]: no need extra space when there is no name 
http://i.imgur.com/6BxmTaB.png

File: src/components/greeting/Greeting.js [hunk 3]
```diff
@@ -0,0 +1,27 @@
+import './greeting.scss';
+
+const calculateGreeting = () => {
+  let greete = '';
+  const hourNow = new Date().getHours();
+  if (hourNow >= 22 || hourNow < 3) {
+    greete = 'Good night';
```
[the remark this answers is not in the document]
Comment (SergeyLinnickHub) [reply src/components/greeting/Greeting.js]: maybe))

File: src/components/greeting/Greeting.js [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+  return greete;
+};
+
+export const Greeting = ({ name }) => (
+  <h2>
+    {calculateGreeting()}
+    {' '}
+    {name}
+
+
```
[the remark this answers is not in the document]
Comment (SergeyLinnickHub) [reply src/components/greeting/Greeting.js]: it happened because of eslint

File: src/components/greeting/Greeting.js (hunk 2 above)
Comment (SergeyLinnickHub) [reply src/components/greeting/Greeting.js]: this is eslint again
need to configure it

**Pull request closed without merging**